SampleProject/DataSource/VOKPersonCollectionDataSource.m

```diff
@@ -0,0 +1,46 @@
+//
+//  VOKPersonCollectionDataSource.m
+//  Vokoder Sample Project
+//
+//  Copyright © 2015 Vokal.
+//
+#import "VOKPersonCollectionDataSource.h"
+#import "VOKCollectionViewCell.h"
+#import "VOKPerson.h"
+
+static NSString *CellIdentifier = @"CellIdentifier";
+
+@implementation VOKPersonCollectionDataSource
+
+- (id)initWithPredicate:(NSPredicate *)predicate
+              cacheName:(NSString *)cacheName
+         collectionView:(UICollectionView *)collectionView
+     sectionNameKeyPath:(NSString *)sectionNameKeyPath
+        sortDescriptors:(NSArray *)sortDescriptors
+     managedObjectClass:(Class)managedObjectClass
+{
+    self = [self initWithPredicate:predicate
+                         cacheName:cacheName
+                    collectionView:collectionView
+                sectionNameKeyPath:sectionNameKeyPath
+                   sortDescriptors:sortDescriptors
+                managedObjectClass:managedObjectClass
+                         batchSize:20];
+    if (self) {
+        CellIdentifier = NSStringFromClass(VOKCollectionViewCell.class);
```

Comment:
Also, if one were to have multiple instances of this data source, they'd clobber each other's `CellIdentifier` values.